document.go

```diff
@@ -0,0 +1,38 @@
+package opslevel
+
+import "github.com/shurcooL/graphql"
+
+type ServiceDocumentSource struct {
+	Integration       `graphql:"... on ApiDocIntegration"`
+	ServiceRepository `graphql:"... on ServiceRepository"`
+}
+
+type ServiceDocument struct {
+	Id         graphql.ID            `graphql:"id" json:"id"`
+	HtmlURL    string                `graphql:"htmlUrl" json:"htmUrl,omitempty"`
+	Source     ServiceDocumentSource `graphql:"source" json:"source"`
+	Timestamps Timestamps            `graphql:"timestamps" json:"timestamps"`
+}
+
+type ServiceDocumentContent struct {
+	ServiceDocument
+	Content string `graphql:"content" json:"content,omitempty"`
```

Comment:
I pulled this out to a dedicated struct so that we are not always pulling back the full contents of the document.  Its currently unused but her incase we need to use it downstream in the CLI or Terraform